helm/prometheus-operator-app/README.md

```diff
@@ -72,45 +75,12 @@ To install the chart with the app CR name `prometheus-operator`:
 
 ### Upgrade
 
-#### From 23.x to 32.x
+#### From 32.x to 42.x
 
 Upgrade the [prometheus-operator-crd](https://github.com/giantswarm/prometheus-operator-crd) chart,
-check Changelog: [23.x to 32.x](../../changelog/23.x_32.x.md)
-
-## Develop
-
-### Upgrade the upstream Chart.
-
```

Comment:
Why was this removed ? It think its helpful

Reply:
I agree but this is a duplicate that is also in the CONTRIBUTING.md file